osf_tests/management_commands/test_move_egap_regs_to_provider.py

```diff
@@ -34,7 +38,7 @@ def egap_reg(self):
         cos = RegistrationProviderFactory(_id='osf')
         return RegistrationFactory(schema=egap_schema, provider=cos)
 
-    def test_move_to_provider(self, egap_provider, egap_reg):
+    def test_move_to_provider(self, egap_provider, egap_reg, non_egap_provider):
         assert egap_reg.provider != egap_provider
 
```

Comment:
You didn't seem to add any assertions about the non_egap_provider to ensure that it was unchanged after the script ran.

Reply:
Ah I see now.

Reply:
ok added here https://github.com/CenterForOpenScience/osf.io/pull/9351/files#diff-5f90a7ad30edf0f466a1c019f7bd9692R53